downstream/modules/platform/proc-create-a-user.adoc

```diff
@@ -1,19 +1,18 @@
 [id="proc-create-a-user_{context}"]
 
-= Creating a User
+= Creating a Keycloak User
 
-This procedure creates a user that can be used to log in to {HubName}.
-As the `hubadmin` role is assigned, this user has Super Administration privileges when logged in.
-
-Use the following procedure to create a Keycloak User.
+Users are entities that are able to log into your system. 
+They can be assigned group membership and have specific roles assigned to them.
+This procedure creates a Keycloak user, with the `hubadmin` role, that can log in to {HubName} with Super Administration privileges.
```

Comment:
I think we can drop the first two concepts because they are overly simplistic and are "Keycloak" concepts. In this procedure all the user cares about is the Keycloak user who is granted admin privileges to Hub. The Keycloak user is an internal user that allows the Operator sufficient access to do the heavylifting for deployment.